src/test/java/RetrofitDeleteBoardTest.java

```diff
@@ -0,0 +1,29 @@
+import okhttp3.ResponseBody;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import retrofit2.Call;
+
+import java.io.IOException;
+
+public class RetrofitDeleteBoardTest extends BaseTestRetrofit{
+
+    @Test
+    public void delete_board() throws IOException {
```

Comment:
This and update, get tests are not atomic, so you could not run them correctly (I mean they would fail probably even if SUT is working correctly) if you have not run create before. It's better to do SUT preparation and clean up in every test. SUT - System Under Test.